src/DynamoCoreWpf/ViewModels/Watch3D/shaderSource/psDynamoMesh.hlsl

```diff
@@ -69,6 +69,11 @@ float4 main(PSInputCustom input, bool isFrontFacing : SV_IsFrontFace) : SV_Targe
     //https://docs.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-semantics
 	 input.n = isFrontFacing ? input.n : -input.n;
 
+	 // if this is a special render package - it should render with the material colors, ambient light
+	 // and not be directionally lit.
+	 if (isSpecialRenderPackage) {
```

Comment:
In line 45 it says
```
IsSpecialRenderPackage = 8, //not used
```
Should we remove the `//not used` part now?

Reply:
thanks @mmisol will do.